mypy/newsemanal/semanal.py

```diff
@@ -4127,3 +4068,21 @@ def apply_semantic_analyzer_patches(patches: List[Tuple[int, Callable[[], None]]
     patches_by_priority = sorted(patches, key=lambda x: x[0])
     for priority, patch_func in patches_by_priority:
         patch_func()
+
+
+def names_defined_by_assignment(s: AssignmentStmt) -> List[str]:
```

Comment:
Although this function is trivial, a short docstring mentioning where this is used will be helpful.